journal-letters/modernized/13-to-barbados.adoc

```diff
@@ -166,30 +164,30 @@ And if he will not hear, nor be gained, then says Christ,
 
 But he gives no commission to Christians to persecute, to put in prison,
 to take away goods, to pull down their houses, to put their feet in the stocks,
-to root them out of the world root and branch; no,
+to root them out of the world, root and branch; no,
 nor to wish it so to be done unto them.
-But if any do, he will reprove such, as he did the disciples, when he told them.
-They knew not what spirit they were of, when they desired fire to come from heaven;
+But if any do, he will reprove such, as he did the disciples, when he told them,
+"`They knew not what spirit they were of,`" when they desired fire to come from heaven;
 for he came not to destroy, but to save, as you may read, Luke 9:54-56.
 And you may see what the apostle Paul says,
-who was a wise master builder; he says.
-One man esteems one day above another, another esteems every day alike; but between them,
-he says.
-Let every man be fully persuaded in his own mind.
+who was a wise master builder; he says,
+"`One man esteems one day above another, another esteems every day alike;`" but between them,
+he says,
+"`Let every man be fully persuaded in his own mind.`"
 So you may see, here is no forcing upon any man,
 in those things that appertain to the worship of God;
 but as Christ overrules the conscience, and persuades the heart,
 and brings man into a belief that it is according to the will of God,
 so that it may be done in the faith, without which none can please God;
-for says the apostle.
-Whatsoever is not of faith, is sin, Rom. 14:5 and 23 verses.
+for says the apostle,
+"`Whatsoever is not of faith, is sin,`" Rom. 14:5, 23.
 Therefore as the gospel is professed, let Christ`'s commands be observed,
 and the example of the primitive Christians followed, who were blessed in their day,
 and had the witness of acceptance with God; who were persecuted,
 but never persecuted any, nor sought to trouble any, as upon a bodily or outward account,
 for their conscience; though they did reprove them sharply,
 that turned away from the power of Truth, and became enemies to the cross of Christ,
-whose belly was their God, who gloried in their shame, and minded earthly things,
+"`whose belly was their God, who gloried in their shame, and minded earthly things,`"
```

Comment:
This one too I kind of think because he changed the tense that I lean away from putting it in quotes. Let's leave them off entirely, I think.

Reply:
ok... I'm bad at quotes... glad you get 'em